src/main/java/com/example/Restaurant_Management_System_REST_API/service/RestaurantCustomerDetailsService.java

```diff
@@ -0,0 +1,27 @@
+package com.example.Restaurant_Management_System_REST_API.service;
+
+import com.example.Restaurant_Management_System_REST_API.model.entity.Customer;
+import com.example.Restaurant_Management_System_REST_API.repository.CustomerRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class RestaurantCustomerDetailsService implements UserDetailsService {
```

Comment:
I'd suggest a change:
"RestaurantCustomerDetailsService" -> "CustomerDetailsService"
It'll be more consistent with entity you created named "Customer" already.

Reply:
OK! great! thanks!